autonomy/analyse/abci/app_spec.py

```diff
@@ -57,6 +87,107 @@ class OutputFormats:  # pylint: disable=too-few-public-methods
         MERMAID = "mermaid"
         ALL = (JSON, YAML, MERMAID)
 
+        default_output_files = {
+            JSON: DEFAULT_FSM_SPEC_JSON,
+            YAML: DEFAULT_FSM_SPEC_YAML,
+            MERMAID: DEFAULT_FSM_SPEC_MERMAID,
+        }
+
+    @classmethod
+    def from_yaml(cls, file: Path) -> Dict:
+        """Load from yaml."""
+        with open_file(file, mode="r", encoding="utf-8") as fp:
+            data = yaml.safe_load(fp)
+
+        validate_fsm_spec(data)
+        return data
+
+    @classmethod
```

Comment:
I'm not going to repeat myself, but we do (mis)use the classmethod decorator unnecessarily often. Let's change that.